http/server.go

```diff
@@ -248,9 +248,10 @@ func (h *DrandHandler) getChainInfo(ctx context.Context, chainHash []byte) (*cha
 
 	bh.chainInfoLk.RLock()
 	if bh.chainInfo != nil {
-		info := bh.chainInfo
+		// we want to return a copy in case it changes
+		info := *bh.chainInfo
```

Comment:
woooahhh I thought `info := bh.chainInfo` was enough to copy the underlying value... turns out I'm wrong. Better go look into my dkg PR again :p